agents/alm.py

```diff
@@ -188,6 +188,11 @@ def _alm_value_loss(self, z_batch, std, log, metrics):
         if log:
             metrics['alm_q_batch'] = Q.mean().item()
         return Q
+    def get_value(self, state):
```

Comment:
You call it as
`self.agent.get_value(state, action)`
but here you do not have action argument